src/core/ledger.js

```diff
@@ -65,22 +61,23 @@ Ledger.prototype.calcOfferEntryHash = function(account, sequence) {
  *
  * The ordering of the two account parameters does not matter.
  *
- * @param {String|UInt160} account1 First Ripple Account
- * @param {String|UInt160} account2 Second Ripple Account
- * @param {String|Currency} currency The currency code
+ * @param {String|UInt160} account1 - First Ripple Account
+ * @param {String|UInt160} account2 - Second Ripple Account
+ * @param {String|Currency} currencyArg - The currency code
  * @return {UInt256}
  */
 Ledger.calcRippleStateEntryHash =
-Ledger.prototype.calcRippleStateEntryHash = function(
-    account1, account2, currency) {
-  currency = Currency.from_json(currency);
-  account1 = UInt160.from_json(account1);
-  account2 = UInt160.from_json(account2);
-
-  if (!account1.is_valid()) {
+Ledger.prototype.calcRippleStateEntryHash = function(account1,
+                                                     account2,
+                                                     currencyArg) {
+  const currency = Currency.from_json(currencyArg);
+  let lowAccount = UInt160.from_json(account1);
+  let hiAccount = UInt160.from_json(account2);
```

Comment:
We don't know that it is the high account yet, so we shouldn't give it this name. I also fixed this in another pull request, here is what I did: https://github.com/ripple/ripple-lib/pull/498/files#diff-d2006f6549215f8372ec508459e5ce44R93